tests/core/creators/duplicates.js

```diff
@@ -7,35 +7,24 @@
 	CKEDITOR.disableAutoInline = true;
 
 	bender.test( {
-		'inline duplicates': function() {
-			var fail = false;
-
-			CKEDITOR.inlineAll();
-
-			wait( function() {
-				try {
-					CKEDITOR.inline( 'editable' );
-					fail = true;
-				} catch ( e ) {}
-
-				assert.isFalse( fail, 'Expected error not thrown.' );
-			}, 100 );
-		},
+		'inline duplicates': createTestCase( 'inline', 'editable' ),
+		'themedui duplicates': createTestCase( 'replace', 'editor' )
+	} );
 
-		'themedui duplicates': function() {
-			var fail = false;
+	function createTestCase( creator, element ) {
+		return function() {
+			var spy = sinon.spy( CKEDITOR, 'error' );
 
-			CKEDITOR.replace( 'editor' );
+			CKEDITOR[ creator ]( element );
 
 			wait( function() {
-				try {
-					CKEDITOR.replace( 'editor' );
-					fail = true;
-				} catch ( e ) {}
+				CKEDITOR[ creator ]( element );
 
-				assert.isFalse( fail, 'Expected error not thrown.' );
+				spy.restore();
+				assert.areSame( 1, spy.callCount, 'Error was thrown' );
+				assert.areSame( 'editor-element-conflict', spy.args[ 0 ][ 0 ], 'Appropriate error code was used' );
```

Comment:
Please, include assertion for `args[ 0 ][ 1 ]` so that it has valid `editorName`. Actually please use `calledWithExactly()` api, in conjuction with custom matcher for `editorName` val it will be nice and readable.

Reply:
Same thing for `element` property in the other `editor-incorrect-element` error.